website/frontend/src/components/blocks/Keyring/Keyring.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect } from 'react';
+import { GearKeyring } from '@gear-js/api';
+import Identicon from '@polkadot/react-identicon';
+
+import './Keyring.scss';
+import { ReadNotificationsIcon } from 'Icons';
+import StatusPanel from 'components/blocks/StatusPanel';
+
+type Props = {
+  handleClose: () => void;
+};
+
+export const Keyring = ({ handleClose }: Props) => {
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  const [key, setKey] = useState('');
+  const [publicKey, setPublicKey] = useState('');
+  const [copySuccess, setCopySuccess] = useState(false);
+  const [isSeed, setIsSeed] = useState('');
+  const [isMnemonic, setIsMnemonic] = useState('');
+  const [saved, setSaved] = useState(false);
+  const [isJson, setIsJson] = useState<any>(null);
+
+  if (copySuccess) {
+    setTimeout(() => setCopySuccess(false), 3000);
+  }
+
+  useEffect(() => {
+    const create = async () => {
+      const { mnemonic, seed, json, json: { address } } = await GearKeyring.create('WebAccount');
+      setKey(mnemonic);
+      setPublicKey(address);
+      setIsSeed(seed);
+      setIsMnemonic(mnemonic);
+      setIsJson(json);
+    };
+
+    create();
+  }, []);
+
+  const copyToClipboard = () => {
+    try {
+      navigator.clipboard.writeText(key);
+      setCopySuccess(true);
+    } catch (err) {
+      setCopySuccess(false);
+    }
+  };
+
+  const downloadJson = (content: any, fileName: string, contentType: string) => {
+    const a: HTMLAnchorElement = document.createElement('a');
+    const file = new Blob([content], { type: contentType });
+    a.href = URL.createObjectURL(file);
+    a.download = fileName;
+    a.click();
+  };
+
+  const handleChange = (event: any) => {
+    if (event.target.value === 'seed') {
+      setKey(isSeed);
+    }
+
+    if (event.target.value === 'mnemonic') {
+      setKey(isMnemonic);
+    }
+  };
+
+  const handleCreate = () => {
+    localStorage.setItem('gear_mnemonic', JSON.stringify(isJson));
```

Comment:
It's better to rename the variable to keyring_json or keyPairJson

Should be checked where it is used

Reply:
I agree.  isJson -> keyPairJson